app/Notifications/Service/ServiceInitialized.php

```
@@ -0,0 +1,80 @@
+                'customer' => $this->service->customer,
+                'product' => $this->service->product,
+                'installationDetails' => $this->installationDetails
+            ]);
+    }
+
+    /**
+     * Get the database representation of the notification.
+     *
+     * @return array<string, mixed>
+     */
+    public function toDatabase(object $notifiable): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$notifiable'. (Unused Code Rules)

(UnusedFormalParameter)